app/models/notification_type.rb

```diff
@@ -49,12 +49,7 @@ def self.seed
     seed_data.each do |t|
       rec = find_by(:name => t[:name])
       t[:expires_in] = t[:expires_in].to_i_with_method
-      if rec.nil?
-        create(t)
-      else
-        rec.update_attributes(t)
-        rec.save!
-      end
+      rec.nil? ? create!(t) : rec.update!(t)
```

Comment:
What do you think of simplifying even further like this?

```ruby
seed_data.each do |t|
  t[:expires_in] = t[:expires_in].to_i_with_method
  rec = find_by(:name => t[:name]) || new
  rec.update!▲
end
```

Reply:
not ▲ but ( t )

Reply:
🔺